package.json

```diff
@@ -41,6 +41,7 @@
     "react-hot-loader": "next",
     "sass-loader": "^6.0.5",
     "style-loader": "^0.18.1",
+    "url-loader": "^0.5.8",
```

Comment:
Is it used in [ webpack configuration](https://github.com/Neufund/generic-ico/blob/basic_layouts/webpack.config.js#L62)

This affects style sheets - [here](https://github.com/Neufund/generic-ico/blob/basic_layouts/src/components/HeaderInitial.scss#L17) 
In reality files if they are small enough (its configurable) they will be embedded into final js file.